pkg/action/install_test.go

```diff
@@ -104,85 +102,15 @@ func TestName(t *testing.T) {
 	}
 	is.Equal("fleet", name)
 
-	// --generate-name while specifying a name
-	instAction = installAction(t)
-	instAction.GenerateName = true
-	chart = buildChart()
-	_, err = instAction.Name(chart, []string{"name1", "chart-uri"})
-	if err == nil {
-		t.Fatal(err)
-	}
-	is.Equal("cannot set --generate-name and also specify a name", err.Error())
-
 	// no name or annotations present
 	instAction = installAction(t)
 	instAction.ReleaseName = ""
 	chart = buildChart()
 	_, err = instAction.Name(chart, []string{"chart-uri"})
```

Comment:
```suggestion
	name, err = instAction.Name(chart, []string{"chart-uri"})
```
If not, we will be doing `is.Equal()` of the previous `name` value.

Reply:
That would explain the bug in Line 113....